app/controllers/admin/beers_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Admin::BeersController < Admin::BaseController
 
   def new
+  #  @s3_direct_post = S3_BUCKET.presigned_post(key: "uploads/#{SecureRandom.uuid}/${filename}", success_action_status: 201, acl: :public_read)
```

Comment:
Line is too long. [143/80]<br>Incorrect indentation detected (column 2 instead of 4).